nautobot/extras/plugins/validators.py

```diff
@@ -40,6 +40,7 @@ def wrap_model_clean_methods():
     """
     Helper function that wraps plugin model validator registered clean methods for all applicable models
     """
-    for model in ContentType.objects.filter(FeatureQuery("custom_validators").get_query()):
-        model_class = model.model_class()
-        model_class.clean = custom_validator_clean(model_class.clean)
+    for app_label, models in FeatureQuery("custom_validators").get_dict():
+        for model in models:
+            model_class = apps.get_model(app_label=app_label, model_name=model)
```

Comment:
Possibly might need/want the `wrap_model_clean_methods` function to take an `apps` parameter (defaulting to the global `apps`) so that for example we could call this function during a migration or a `nautobot_database_ready` signal handler where `apps` is a different context? Not sure, just thinking aloud.

Reply:
@glennmatthews your thought is so if a custom validator is available we can use it in migrations now? Because I assume that doesn't work at present?

Reply:
Yeah, seems pretty clear it wouldn't work at present. We *probably* don't want them to apply during migrations in general (that would be troublesome) but I feel like there *might* be a use case for having them apply during the database-ready callback signals? As I said, not sure. :-)

Reply:
Required for this PR or future housekeeping/feature?

Reply:
Definitely not required.